npm-debug.log

```diff
@@ -1,107 +0,0 @@
-0 info it worked if it ends with ok
```

Comment:
Well, very silly of me to let this get in here. Need a global .gitignore ... Doh!